playbook.yaml

```diff
@@ -0,0 +1,56 @@
+---
+- name: install and start todo app
+  hosts: frontend
+  remote_user: ec2-user
+  vars_prompt:
+    - name: secret_key
+      prompt: Flask secret key?
+      private: false
```

Comment:
This is fine with how we've currently got the system set up, but it would be worth switching to `private: true` regardless on a real system - it's easy to imagine someone taking this playbook and switching to have it run on a pipeline, and then to discover Ansible is logging away secrets that we definitely don't want exposed on our CI server!

Reply:
Good point, not sure why I left this as private actually..